src/pages/Detail/Component/Review.js

```diff
@@ -0,0 +1,260 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { FaStar } from 'react-icons/fa';
+import { FaTrashAlt } from 'react-icons/fa';
+
+const Review = ({ bookData, setBookData }) => {
+  const stars = Array(5).fill(0);
+  const [review, setReview] = useState([]);
+  const [hoverValue, setHoverValue] = useState(undefined);
+  const [newReview, setNewReview] = useState({
+    book_id: 1,
+    review: '',
+    rating: 0,
+  });
+
+  const isValid = newReview.review.length >= 5 && newReview.rating >= 1;
+
+  const clickStarRate = value => {
+    setNewReview(prev => {
+      return { ...prev, rating: value };
+    });
+  };
+
+  const reviewInputHandler = e => {
+    setNewReview(prev => {
+      return { ...prev, review: e.target.value };
+    });
+  };
```

Comment:
이 부분도 단순 return이라면 () => () 형태로 만들 수 있겠네요!
```suggestion
  const clickStarRate = value => {
    setNewReview(prev => ({
      ...prev, rating: value
    }));
  };

  const reviewInputHandler = e => {
    setNewReview(prev => ({
      ...prev, review: e.target.value
    }));
  };
```